pulp_ansible/app/galaxy/views.py

```diff
@@ -154,10 +154,7 @@ def post(self, request, path):
         )
         serializer.is_valid(raise_exception=True)
 
-        expected_digests = {"sha256": serializer.validated_data["sha256"]}
-        artifact = Artifact.init_and_validate(
-            serializer.validated_data["file"], expected_digests=expected_digests
```

Comment:
why we are not validating the checksum anymore?

Reply:
We can't anymore because the ansible-galaxy CLI doesn't send it. If we continue to require it then user's couldn't upload for many months since ansible 2.9 would be broken and 2.10 isn't coming out for a long time.